cmd/markdown/atmos_toolchain_lock.md

```
@@ -0,0 +1,15 @@
+
+- Lock every tool in .tool-versions
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)